broker/traffic_control.py

```diff
@@ -41,3 +41,6 @@ def set_fixed_bandwidth(self, bandwidth):
         self.tc('class add dev %s parent 1: classid 1:1 htb rate %dkbit ceil %dkbit' % (
             self.interface, bandwidth, bandwidth
         ))
+        self.tc('tc qdisc add dev %s parent 1:1 fq_codel' % (
+            self.interface)
+        ))
```

Comment:
There is an extra closing brace (also why the Travis tests fail). BTW, can you also use `ignore_fails=True`, so that in case `fq_codel` is not supported, the configuration will not fail?